bootstrap/components/sources/hdfs-source-topology-component.json

```diff
@@ -33,6 +33,24 @@
         "defaultValue": "/tmp/sourcefiles/",
         "hint": "schema"
```

Comment:
@satishd I assume the user will be using newly added `Schema Name` field to get schema data so I assume this hint is no longer needed.

@priyank5485 Your thoughts?

Reply:
@shahsank3t I am trying to understand the existing config here. Does that indicate to get schema using that name from registry?

Reply:
@satishd As discussed/explained earlier, you need to remove hint from here, rest of the changes looks good.